README.md

```diff
@@ -13,19 +13,60 @@ Compilation has been tested with:
 Underlying libraries:
 - [ZLib](http://zlib.net) 1.2.11
 - [libzip](http://www.nih.at/libzip) 1.5.2
+- BZip2
```

Comment:
Do we have to download this library alongside with zlib and libzip ? That would need some explanation in the readme :)

Reply:
Depends. It seems BZip2 is only a link-dependency of libzip. `compile.bat` uses ENABLE_BZIP2=OFF, but FindLIBZIP requires it anyway. Also it is not the default, so I guess it makes more sense to use it. It is nothing new so maybe do it in a separate PR?

Reply:
@Flamefire sorry to get back so late, December was really busy :sweat_smile: Happy New Year BTW !

Let's get this merged, can you just add the link to BZip2 and is it possible to have a documentation on how to install it manually in order for FindLIBZIP to find it (especially on Windows) ? Thanks !

Reply:
@Flamefire ping :sweat_smile:

Reply:
Added the link.

Install BZip2 on windows is no fun and actually it is optional. I changed the logic to reflect that